src/services/scenario/ScenarioService.js

```diff
@@ -37,9 +39,40 @@ function createScenario (organizationId, workspaceId, scenario) {
   });
 }
 
-function updateAndLaunchScenario (organizationId, workspaceId, scenariodId, scenarioParameters) {
+function formatScenarioParametersForApi (scenarioParameters) {
+  // Reformat scenario parameters to match the API expected types
+  return scenarioParameters.map(param => {
+    // Clone the original parameter
+    const newParam = clone(param);
+    // Cast boolean values into string values
+    if (newParam.varType === 'bool') {
+      newParam.value = newParam.value.toString();
+    }
+    return newParam;
+  });
+}
+
+function formatScenarioParametersFromApi (scenarioParameters) {
```

Comment:
Should be placed in utility file.
We will certainly reuse the logic here somewhere else :thinking: